composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
@jmikola The project you mention has probably much less attention on less used versions. But I understand you're not into semantic versioning argument.